src/pages/gd/Claim/index.tsx

```diff
@@ -3,30 +3,47 @@ import { t } from '@lingui/macro'
 import { useLingui } from '@lingui/react'
 // TODO: check when good-design will be merged
 import { ClaimButton, Layout, Title } from '@gooddollar/good-design'
-import { StyleSheet } from "react-native";
-import { View } from "native-base";
+import { StyleSheet } from 'react-native'
+import { View, Text } from 'native-base'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
 
 const Claim = () => {
-  const { i18n } = useLingui()
-
-  return (
-    <Layout>
-      <Title>
-        {i18n._(t`Claim`)}
-      </Title>
-      <View style={styles.content}>
-        <ClaimButton  firstName={'Test'} method={'redirect'}/>
-      </View>
-    </Layout>
-  )
+    const { i18n } = useLingui()
+    const { account } = useActiveWeb3React()
+    return (
```

Comment:
```suggestion

    return (
```
pls add empty lines betweek code blocks, this increases readability